pypeit_files/keck_deimos_830g_m_8500.pypeit

```diff
@@ -7,6 +7,8 @@
 [baseprocess]
    use_biasimage=False
 [reduce]
+   [[findobj]]
+         sig_thresh = 100.
```

Comment:
This is a very high value. In general this is not recommended since objects that don't get traced don't get masked, and thus the sky background is biased by them.

Reply:
This is set on purpose to check if `deimos_force_extract` works properly. To do so, I need to put a high `sig_thresh` so that I can miss some obvious targets and then check if they are recovered.

Reply:
Got it. Please just add a comment to this pypeit file explaining that, so that some user does not think this is a wise thing to do.

Reply:
Done